deploys/classdemo/terraform/main.py

```diff
@@ -0,0 +1,30 @@
+"""Demonstrate a synth function that uses a subclass of HeliStack."""
+
+from cdktf import LocalExecProvisioner, TerraformLocal, TerraformOutput, TerraformVariable
+from cdktf_cdktf_provider_null.resource import Resource as NullResource
+
+from helicopyter import HeliStack
+
+
+class NewStack(HeliStack):
+    """Infer the ENVIronment (ENVI) from the workspace and provide it and cona as locals."""
+
+    def __init__(self, cona: str) -> None:
+        super().__init__(cona)
+        self.push(TerraformLocal, 'cona', self.cona)
+        self.push(TerraformLocal, 'envi', '${terraform.workspace}')
+
+
+def synth(stack: NewStack) -> None:
+    """Accept the Git hASH (GASH) as a variable and output it."""
```

Comment:
Imo I would not associate GASH with git hash, why not just `git_hash`?

Reply:
Yeah, the Four Letter Abbreviations (FLANs) are weird. One upside is that they can be given very specific definitions. Maybe that's not truly needed. Another upside is tidy display in space-constrained circumstances.

https://github.com/biobuddies/helicopyter/blob/59b6ee6b0ee1be4ba665fef1ff8929eed909a892/style.md?plain=1#L79-L84
https://github.com/biobuddies/helicopyter/blob/59b6ee6b0ee1be4ba665fef1ff8929eed909a892/includes.sh#L352-L368

Perhaps this is me over-correcting from a 17-character equivalent to `cona`, and the SignalFx dimension display that is tiny for no good reason.

I'd like to keep experimenting with them for a while longer.